nexus/types/src/deployment.rs

```diff
@@ -461,14 +470,17 @@ pub enum BlueprintZoneFilter {
     /// All zones.
     All,
 
-    /// Filter by zones that should be in internal DNS.
-    InternalDns,
+    // Zones that are desired to be in the RUNNING state
```

Comment:
```suggestion
    /// Zones that are desired to be in the RUNNING state
```

Reply:
Done in [81df651](https://github.com/oxidecomputer/omicron/pull/5348/commits/81df6517be952e5baf1500135f34cf7e6d19ebcc)